### ISSUE TITLE
Generic App names

### ISSUE THREAD

**Issue by Foggalong** [ozonos/atom-launcher#13]
We need generic apps name. "Chromium" - "Web Browser", "gThumb" - "Image Viewer".

(I know this doesn't come under the remit of the launcher extension but posting it here in lieu of a better place I'm filling it here)

**Comment by me4oslav** [comment]
Yeah. I guess we can sed that stuff, right?

**Comment by Foggalong** [comment]
Yeah, it should simply be a case of changing the `Name` line in the launchers that we ship

**Comment by me4oslav** [comment]
Good. Are there legal issues?

**Comment by Foggalong** [comment]
Ah, I'd not thought about that. Probably not with a lot of them, but possibly with Chromium. Might be worth emailing their dev team to ask

**Comment by cldx** [comment]
Why would we even change Chromium? I can understand it for things like gthumb which are just horrible names. But Chrome is a well known brand.

**Comment by me4oslav** [comment]
@cldx I guess "Chromium" is ok, but it says "Chromium Web Browser" which at the launcher looks like "Chromium We..." which as you can imagine is not pretty.

**Comment by Foggalong** [comment]
Yeah, I think changing Chromium to 'web browser' would be as beneficial as the rest of them. The only program I think wouldn't benefit would be Steam because it's completely useless without a Steam account :P

**Comment by Foggalong** [comment]
IIRC distros usually have "generic names" as a toggle in the settings too so if people are more comfortable with searching that way they can

**Comment by me4oslav** [comment]
So, has anyone written some sort of script or something for that?
